operations/validate_csm_health.md

```diff
@@ -116,21 +116,7 @@ See [Known issues with NCN resource checks](../troubleshooting/known_issues/ncn_
 
 ### 1.3 Check of system management monitoring tools
 
-If all designated prerequisites are met, the availability of system management health services may optionally be validated by accessing the URLs listed in
-[Access System Management Health Services](system_management_health/Access_System_Management_Health_Services.md).
-It is very important to check the `Prerequisites` section of this document.
-
-If one or more of the the URLs listed in the procedure are inaccessible, it does not necessarily mean that system is not healthy. It may simply mean that not all of the
-prerequisites have been met to allow access to the system management health tools via URL.
-
-Information to assist with troubleshooting some of the components mentioned in the prerequisites can be accessed here:
-
-- [Troubleshoot CMN Issues](network/customer_accessible_networks/Troubleshoot_CMN_Issues.md)
-- [Troubleshoot DNS Configuration Issues](network/external_dns/Troubleshoot_DNS_Configuration_Issues.md)
-- [Check BGP Status and Reset Sessions](network/metallb_bgp/Check_BGP_Status_and_Reset_Sessions.md)
-- [Troubleshoot BGP not Accepting Routes from MetalLB](network/metallb_bgp/Troubleshoot_BGP_not_Accepting_Routes_from_MetalLB.md)
-- [Troubleshoot Services without an Allocated IP Address](network/metallb_bgp/Troubleshoot_Services_without_an_Allocated_IP_Address.md)
-- [Troubleshoot Prometheus Alerts](system_management_health/Troubleshoot_Prometheus_Alerts.md)
+In order to validate any issue with NCN please run [ncn-csm-monitoring-validation](https://github.com/Cray-HPE/csm-testing/blob/40619bcee8bdedb113341b9c8ad7817fb8cbf8b1/goss-testing/automated/ncn-csm-monitoring-validation) script from csm-testing repo
```

Comment:
We should not be linking to the csm-testing repository on github. Look at how the other Goss tests are run earlier in this same section of the healthchecks. The instructions for these should be similar.